### PR TITLE
SIP 4.19.3 handles ctor call as default value of an argument

### PR DESCRIPTION
**Pull request by 3nids**
https://www.riverbankcomputing.com/hg/sip/rev/5df8870c61a7

## Description
Include a few sentences describing the overall goals for this PR (pull request). If applicable also add screenshots.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: python/core/qgsrange.sip [hunk 1]
```diff
@@ -147,15 +147,19 @@ class QgsTemporalRange
 %End
   public:
 
-    QgsTemporalRange( const T &begin, const T &end, bool includeBeginning = true, bool includeEnd = true );
+#if SIP_VERSION >= 0x41303
 %Docstring
  Constructor for QgsTemporalRange. The ``begin`` and ``end`` are specified,
  and optionally whether or not these bounds are included in the range.
```
Comment (m-kuhn) [inline python/core/qgsrange.sip]: Can you rephrase that to something like

``begin`` and ``end`` can optionally be passed to specify if bounds are included in the range.

Comment (3nids) [inline python/core/qgsrange.sip]: nope, it's not exactly what was meant here (by @nyalldawson). It's includeBegining/End which specify if bounds are included in the range. begin and end are optional in cpp, mandatory in python (SIP<4.19.3).

Comment (m-kuhn) [inline python/core/qgsrange.sip]: > are specified, and optionally whether...

still doesn't sound like proper English to me ;)
